04_UpdateUser.php

```diff
@@ -2,106 +2,104 @@
+		<meta name = "Author" content = "Liv Amer"/>
+		<meta name = "Description" content = "Graeme's Music Database"/>
+		<meta name = "viewport" content = "width=device-width,initial-scale=1"/>
+
 
 	</head>
-		<body>
-		
-			<!-- Navbar (sit on top) -->
-			<div class="w3-top">
-			 	<div class="w3-bar w3-white w3-wide w3-padding w3-card">
-					<img src="images/placeholder.jpg" width="40" height="40">
-					<a href="index.html" class="w3-bar-item w3-button"><b>logo</b> </a>
-	  
-		 <!-- Float links to the right -->
-					<div class="w3-right" >
-					  <a href="indexV1.php" class="w3-bar-item w3-button w3-xxlarge fa fa-home"></a>
-					  <a href="playlistV1.php" class="w3-bar-item w3-button w3-xxlarge material-icons">playlist_add</a>	
-					  <a href="contact.php" class="w3-bar-item w3-button w3-xxlarge material-icons" >contact_support</a>
-					  <a href="settings.php" class="w3-bar-item w3-button w3-xxlarge material-icons">settings</a>	
-    				</div>
+	<body>
+
+		<!-- Navbar (sit on top) -->
+		<div class="w3-top">
+			<div class="w3-bar w3-grey w3-padding w3-card">
+				<img src="images/placeholder.jpg" width="40" height="40">
+				<a href="indexV1.php" class="w3-bar-item w3-button"><b>Soundwave</b> </a>
+
+				<!-- Float links to the right -->
+				<div class="w3-right" >
+					<a href="indexV1.php" class="w3-bar-item w3-button w3-xxlarge fa fa-home"></a>
+					<a href="playlistV1.php" class="w3-bar-item w3-button w3-xxlarge material-icons">playlist_add</a>	
+					<a href="contact.php" class="w3-bar-item w3-button w3-xxlarge material-icons" >contact_support</a>
+					<a href="settings.php" class="w3-bar-item w3-button w3-xxlarge material-icons">settings</a>	
 				</div>
 			</div>
-		
-		
+		</div>
+
+
 		<br>
 		<br>
 		<br>
 
 
-			<!-- side navbar for admin + php codes -->
-			<div class="w3-sidebar w3-bar-block w3-border-right" style="width:8%">
-				<a href="03_add_user(settings).php" class="w3-bar-item w3-button w3-xxlarge material-icons">person_add</a> 
-				<a href="02_show_user.php" class="w3-bar-item w3-button w3-xxlarge material-icons">visibility</a> 
-			 	<a href="04_update_user.php" class="w3-bar-item w3-button w3-xxlarge material-icons">manage_accounts</a>
-			  	<a href="05_delete_user.php" class="w3-bar-item w3-button w3-xxlarge material-icons">person_remove</a>
+		<!-- side navbar for admin + php codes -->
+		<div class="w3-sidebar w3-bar-block w3-border-right" style="width:8%">
+			<a href="03_add_user(settings).php" class="w3-bar-item w3-button w3-xxlarge material-icons">person_add</a> 
+			<a href="02_show_user.php" class="w3-bar-item w3-button w3-xxlarge material-icons">visibility</a> 
+			<a href="04_update_user.php" class="w3-bar-item w3-button w3-xxlarge material-icons">manage_accounts</a>
+			<a href="05_delete_user.php" class="w3-bar-item w3-button w3-xxlarge material-icons">person_remove</a>
 
+		</div>
+
+
+
+		<div style="margin-left:8%">
+			<div class="w3-container w3-dark-grey">
+				<h2>Update User</h2>
```

Comment:
changed title for admin pages to h2 = big white title text so Graeme can see it clearly; readability